src/eduid_webapp/security/views/webauthn.py

```diff
@@ -123,13 +124,22 @@ def registration_complete(user, credential_id, attestation_object, client_data,
 @require_user
 def remove(user, credential_key):
     security_user = SecurityUser.from_user(user, current_app.private_userdb)
-    token_to_remove = security_user.credentials.filter(Webauthn).find(credential_key)
+    tokens = security_user.credentials.filter(FidoCredential)
+    if tokens.count <= 1:
+        return {'_error': True, 'message': 'security.webauthn-noremove-last'}
+    token_to_remove = security_user.credentials.find(credential_key)
     if token_to_remove:
         security_user.credentials.remove(credential_key)
         save_and_sync_user(security_user)
         current_app.stats.count(name='webauthn_token_remove')
+        current_app.logger.info(f'User {security_user} has removed a security token: {credential_key}')
+        message = 'security.webauthn-token-removed'
+    else:
+        current_app.logger.info(f'User {security_user} has tried to remove a'
+                                 ' missing security token: {credential_key}')
+        message = 'security.webauthn-token-notfound'
```

Comment:
don't know what is really necessary, but pycharm wants the second line of a split string to also have the "f"